src/DwitTech.AccountService.Core/Utilities/RandomUtil.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Utilities
+{
+    public static class RandomUtil 
+
+    {
+        internal static readonly char[] chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890".ToCharArray();
+        public static string GenerateUniqueCode(int numberOfCharacters=20, bool useNumbers = true, bool useAlphabets = true, bool useSymbols = false)
```

Comment:
the boolean parameters here dont seem to be in use. So why are they there?

Reply:
They're there because I was a little bit confused as to what to do with them since you gave me the format. Should I use if statements to tweak the seed values based on the parameter values, or can I delete them?

Reply:
Ah ah. So you know u can use if statements to tweak the seed values

Reply:
Lol. Ok, I'll do that